### DISCUSSION TITLE
AssertionError: ERROR: string mismatch between PM7 on reference line 10 and Non on output line 10

### DISCUSSION THREAD

**Discussion by 123xyl** [openmopac/mopac#127] [Q&A]
when ctest 
![image](https://user-images.githubusercontent.com/96686347/194715875-7497516a-5c32-428e-9acb-b7e8c8ceefd3.png)

**Comment by godotalgorithm** [comment]
You previously reported mostly successful tests, so you'll need to clarify what you are doing differently now. The testing error messages are only summarizing the differences between the output files of the tests in the `build/tests` subdirectory and the reference data in `tests` subdirectory. It would help to see a pair of output files that are failing the tests. The small size noted by the output file size mismatch suggest that MOPAC is producing an error message and terminating prematurely.
